apps/jupyter/extensions/notebook-runner/src/iframe-api.ts

```diff
@@ -407,6 +407,67 @@ export class EmbeddedPythonCallbacks {
       throw new Error(`Unsupported content format: ${file.format}`);
     }
   }
+
+  private async writeFolderContents(
+    basePath: string,
+    files: Map<string, Blob>,
```

Comment:
I think having a custom type alias would go a long way for all of these methods :) (easier to read and change should we ever need to)

Reply:
has been addressed by @bubudetp